scap-security-guide.spec.in

```diff
@@ -89,8 +89,13 @@ cp -a RHEL/6/dist/content/ssg-rhel6-ds.xml %{buildroot}%{_datadir}/xml/scap/ssg/
 cp -a RHEL/7/dist/content/ssg-rhel7-ds.xml %{buildroot}%{_datadir}/xml/scap/ssg/rhel7
 %endif
 
+# Add in oscap-scan files
+mkdir -p %{buildroot}/etc/sysconfig/
+cp -a {*,*/*}/input/oscap-scan-* %{buildroot}/etc/sysconfig/
```

Comment:
Since its within the project folder and only captures patterns matching
/input/oscap-scan-*, it seemed rather harmless. It also seems much simpler
that explicitly indicating each existing path, and allows for ease of
maintaining when other project and/or oscap-scan files are added. However,
I can change it to explicitly add each path, if required.

On Fri, Feb 27, 2015 at 10:29 AM redhatrises <notifications@github.com>
wrote:

> In scap-security-guide.spec.in
> <https://github.com/OpenSCAP/scap-security-guide/pull/465#discussion_r25518178>
> :
>
> > @@ -89,8 +89,13 @@ cp -a RHEL/6/dist/content/ssg-rhel6-ds.xml %{buildroot}%{_datadir}/xml/scap/ssg/
> >  cp -a RHEL/7/dist/content/ssg-rhel7-ds.xml %{buildroot}%{_datadir}/xml/scap/ssg/rhel7
> >  %endif
> >
> > +# Add in oscap-scan files
> > +mkdir -p %{buildroot}/etc/sysconfig/
> > +cp -a {*,*/*}/input/oscap-scan-* %{buildroot}/etc/sysconfig/
>
> Probably shouldn't do cp -a {*,*/*}.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSCAP/scap-security-guide/pull/465/files#r25518178>
> .
>